src/Components/BookPage/BookSynopsis.js

```diff
@@ -1,65 +1,467 @@
+    border: '0.5px solid #E8E8E8',
+    '@media (max-width: 960px)': {
+      width: '85%',
+      margin: 'auto',
+    },
+  },
+  sideCardTitle: {
+    textAlign: 'left',
+    fontFamily: 'Work Sans',
+    fontSize: '1.12em',
+    fontWeight: '600',
+    lineHeight: '1.5em',
+    color: '#020202',
+    marginTop: '2vh',
+  },
+  sideCardLinkContainer: {
+    textAlign: 'left',
+    fontSize: '0.85em',
+    margin: '2vh auto 2vh auto',
+  },
+  creator: {
+    display: 'inline',
+    fontFamily: 'DM Sans',
+    color: '#5A5A5A',
+    lineHeight: '1.3em',
+  },
+  chip: {
+    margin: '0.2vw',
+    fontSize: '0.9em',
+    fontFamily: 'DM Sans',
+  },
+  bolded: {
+    textAlign: 'left',
+    fontFamily: 'Work Sans',
+    fontSize: '1.12em',
+    fontWeight: '600',
+    lineHeight: '0',
+  },
+  blockContainer: {
+    margin: '1vh auto',
+    border: '1vh 1vw 1vh 1vh',
+    display: 'flex',
+    flexDirection: 'row',
+    marginTop: '4vh',
+  },
+  block: {
+    fontFamily: 'DM Sans',
+    width: '45%',
+    textAlign: 'left',
+    flex: 'none',
+    order: '1',
+    flexGrow: '0',
+
+  },
+  sub: {
+    lineHeight: '1.5em',
+    color: '#656565',
+  },
+  bookshop: {
+    fontFamily: 'Work Sans',
+    fontWeight: '600',
+    color: '#3477DE',
+    background: 'rgba(52, 119, 222, 0.06)',
+  },
+  linkUI: {
+    textDecoration: 'none',
+    color: '#3477DE',
+    fontWeight: '700',
+    lineHeight: '1.8em',
+    fontSize: '1.05em',
+  },
+  link: {
+    textDecoration: 'none',
+    color: 'royalblue',
+    fontSize: '1.15em',
+  },
+  seeMoreContainer: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  seeMoreText: {
+    fontFamily: 'DM Sans',
+    fontWeight: 700,
+    fontSize: '0.875em',
+    color: '#0068C9',
+    paddingTop: 1,
+    '&:hover': {
+      color: '#669afa',
+      cursor: 'pointer',
+    },
+  },
+  seeMoreIcon: {
+    fontSize: '1em',
+    color: '#0068C9',
+    paddingTop: 1,
+    paddingLeft: 0.25,
+    '&:hover': {
+      color: '#669afa',
+      cursor: 'pointer',
+    },
+  },
+
+};
+
+const chipColors = [
+  { // blue
+    color: '#004A90',
+    backgroundColor: '#D0E6FB',
+  },
+  { // red
+    color: '#A51717',
+    backgroundColor: '#FFDFDF',
+  },
+  { // green
+    color: '#006A63',
+    backgroundColor: '#C1F0ED',
+  },
+];
+
+/* Grade Range
+    Airtable data for grade range is not dev-friendly. 😔
+    if either one of gradeMin and gradeMax are undefined, we don't render at all.
+    This does not handle Airtable user errors such as
+      gradeMin: 9th, gradeMax: Kindergarten
+    where the order of grades are incorrect
+  */
+const createGradeRange = (gradeMin, gradeMax) => {
+  let gradeRange;
+  if (!gradeMin) {
+    // gradeMin not given
+    gradeRange = gradeMax;
+  } else if (!gradeMax) {
+    // gradeMax not given
+    gradeRange = gradeMin;
+  } else if (gradeMin === '0 to Pre-K' && gradeMax.length < 5) {
+    // 0 to Pre-K to short grade
+    gradeRange = `0 to ${gradeMax}`;
```

Comment:
can you change it to "Up to ${gradeMax}", 0 doesn't make much sense in a grade context
```suggestion
    gradeRange = `Up to ${gradeMax}`;
```